server/services/userService.ts

```diff
@@ -30,4 +33,28 @@ export default class UserService {
       deliusUsername: communityAccommodationUser.deliusUsername,
     }
   }
+
+  getActingUserPrimaryNavigationList(actingUser: User, currentPage: string): Array<PrimaryNavigationItem> {
+    if (actingUser.roles?.includes('assessor')) {
+      const navList: Array<PrimaryNavigationItem> = [
+        { href: paths.bookings.index({}), text: 'Bookings' },
+        { href: paths.premises.index({}), text: 'Manage properties' },
+        { href: paths.assessments.index({}), text: 'Referrals' },
+        { href: paths.bedspaces.search({}), text: 'Search bedspaces' },
+        { href: paths.reports.index({}), text: 'Reports' },
+      ]
+
+      navList.every(item => {
+        if (currentPage.includes(item.href)) {
+          item.active = true
+          return false
+        }
+        return true
+      })
+      return navList
```

Comment:
This feels odd -- `every` is typically used to assert every element meets a condition. Could this use `map` instead?

```suggestion
      return navList.map(item => ({
        ...item,
        active: currentPage.includes(item.href) || undefined, // or leave this as `false` instead of undefined
      }))
```